tests/test_storage.py

```diff
@@ -0,0 +1,40 @@
+from pathlib import Path
+
+import h5py
+
+from lamindb.dev.storage import _infer_filesystem, delete_storage
+from lamindb.dev.storage._backed_access import AnnDataAccessor
+from lamindb.dev.storage._file import read_adata_h5ad
+from lamindb.dev.storage._zarr import read_adata_zarr, write_adata_zarr
+
+
+def test_anndata_io():
+    test_files = Path("tests/test-files")
+
+    adata = read_adata_h5ad(test_files / "pbmc68k.h5ad")
```

Comment:
See our Slack discussion.

This file should be pulled from S3 in a fixture! ☺️